openhands_ext/test_multiuser.py

```
@@ -0,0 +1,125 @@
+# Test extension router
+router = APIRouter(prefix="/test-multiuser")
+security = HTTPBearer()
+
+# Simple in-memory user store (enterprise would use database)
+USERS = {
+    "user1": {"id": "user1", "email": "user1@example.com", "plan": "free"},
+    "user2": {"id": "user2", "email": "user2@example.com", "plan": "pro"},
+}
+
+# JWT secret (enterprise would use proper key management)
+JWT_SECRET = "test-secret-key"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "JWT_SECRET"

(S105)